packages/nox/src/NOX_Solver_SingleStep.C

```diff
@@ -132,7 +133,7 @@ bool NOX::Solver::SingleStep::try_step()
   Teuchos::RCP<NOX::Abstract::Group> jacobian = updateJacobian ? solnPtr : frozenJacobianPtr;
 
   // Reuse memory in group instead of new allocation for dir
```

Comment:
Changes invalidate this comment. Could you remove the comment above?

Reply:
Sure, I missed that. Done.